src/Illuminate/Testing/TestResponse.php

```diff
@@ -692,13 +692,13 @@ public function assertJsonMissingValidationErrors($keys = null, $responseKey = '
 
         $json = $this->json();
 
-        if (! array_key_exists($responseKey, $json)) {
-            PHPUnit::assertArrayNotHasKey($responseKey, $json);
+        if (! Arr::has($json, $responseKey)) {
+            PHPUnit::assertFalse(Arr::has($json, $responseKey));
```

Comment:
This is going to make the error message much more vague because you changed this to a plain assertFalse with no additional message information.

Reply:
Updated.